WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -42,16 +42,6 @@ public WabiSabiConfig(string filePath) : base(filePath)
 		[JsonConverter(typeof(MoneyBtcJsonConverter))]
 		public Money MaxRegistrableAmount { get; set; } = Money.Coins(43_000m);
 
-		/// <summary>
-		/// How many virtual bytes the server gives to alices per registrations.
-		/// If it's 250, then about 400 alices can participate.
-		/// The width of the rangeproofs are calculated from this, so don't choose stupid numbers.
-		/// Consider that it applies to registrations, not for inputs. This usually consists one input, but can be more.
```

Comment:
maybe worth preserving this information next to `MaxInputCountByRound`? (aside: multiple outputs is out of date, my bad)